### PR TITLE
VEIOSS-435 - Implement multiple payment screen

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: test/uk/gov/hmrc/iossreturns/connectors/RegistrationConnectorSpec.scala [hunk 1]
```diff
@@ -24,14 +24,24 @@ class RegistrationConnectorSpec extends SpecBase with WireMockHelper {
...
+           | "exclusions": ${Json.toJson(etmpRegistration.exclusions)},
+           | "adminUse": ${Json.toJson(etmpRegistration.adminUse)}
+           |}
+           }""".stripMargin
       running(application) {
 
         val connector: RegistrationConnector = application.injector.instanceOf[RegistrationConnector]
 
-        val responseBody = Json.toJson(etmpRegistration).toString()
+        //        val responseBody = Json.toJson(etmpRegistration).toString()
```
Comment (pbyrne84) [inline test/uk/gov/hmrc/iossreturns/connectors/RegistrationConnectorSpec.scala]: Commented out line

File: app/uk/gov/hmrc/iossreturns/services/PaymentsService.scala [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import uk.gov.hmrc.iossreturns.utils.Formatters._
+
+import java.time.LocalDate
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class PaymentsService @Inject()(
+                                 financialDataConnector: FinancialDataConnector,
+                                 vatReturnConnector: VatReturnConnector//,
+                                 //paymentConnector: PaymentConnector
```
Comment (hmrcvmu) [inline app/uk/gov/hmrc/iossreturns/services/PaymentsService.scala]: leftover comment

File: test/uk/gov/hmrc/iossreturns/services/PaymentsServiceSpec.scala [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+import uk.gov.hmrc.iossreturns.models.financialdata.{FinancialData, FinancialTransaction, Item}
+import uk.gov.hmrc.iossreturns.models.payments.Payment
+
+import java.time._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class PaymentsServiceSpec extends SpecBase with MockitoSugar with BeforeAndAfterEach with PaymentsServiceSpecFixture with ScalaCheckPropertyChecks {
+  implicit val hc = HeaderCarrier()
+//  val paymentConnector = mock[PaymentConnector]
```
Comment (hmrcvmu) [inline test/uk/gov/hmrc/iossreturns/services/PaymentsServiceSpec.scala]: leftover comment

File: app/uk/gov/hmrc/iossreturns/services/PaymentsService.scala [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+      status = "A"
+    )
+
+    val financialDataQueryParameters: FinancialDataQueryParameters = FinancialDataQueryParameters(
+      fromDate = Some(beginningOfTime),
+      toDate = Some(now)
+    )
+
+    for {
+      financialData <- getFinancialData(iossNumber, financialDataQueryParameters) //Todo: Recheck
```
Comment (pbyrne84) [inline app/uk/gov/hmrc/iossreturns/services/PaymentsService.scala]: Todo?

**Review by hmrcvmu — APPROVED** [review]

**Pull request merged**